docs/syscall_descriptions.md

```diff
@@ -84,6 +84,9 @@ Note: _all_ generated files (`*.const`, `*.go`, `*.h`) are checked-in with the
 Note: `make extract` extracts constants for all architectures which requires
 installed cross-compilers. If you get errors about missing compilers/libraries,
 try `sudo make install_prerequisites` or install equivalent package for your distro.
+The command install_reqrequisites will success even some of the installation failed,
+sudo apt-get update && sudo apt-get upgrade might be required to make this more
+efficient. Also you will want to add $GOPATH/bin to your PATH environment variable.
```

Comment:
Please mention GOPATH here:
https://github.com/google/syzkaller/blob/master/docs/contributing.md#go
We can't duplicate all instructions everywhere.
We could mention somewhere in the beginning of this document that one needs a working environment with a link to the setup instructions, but duplicating parts of the instructions throughout is not a good idea.

Reply:
Yes that make sense, the $GOPATH/bin should be required for presubmit mentioned at contributing.md. Moving this over